ios/FluentUI.Demo/FluentUI.Demo/Demos/DrawerDemoController.swift

```diff
@@ -474,6 +474,12 @@ class DrawerDemoController: DemoController {
         drawer.contentScrollView = personaListView
     }
 
+    private class CustomResizingHandleTokens: ResizingHandleTokens {
```

Comment:
Is this class being used anywhere?

Reply:
Ah. No. I meant to remove it, but missed it. I'll be sure to clean it up when I resolve the other TODO